src/Config.js

```diff
@@ -453,10 +455,19 @@ export class Config {
     }
   }
 
-  static validateMaxLimit(maxLimit) {
+  static validateDefaultLimit(defaultLimit) {
+    if (defaultLimit <= 0) {
```

Comment:
Could you take a look at look at the more modern `validateFileUploadOptions` example. There are multiple parts to validation:
1. is the value of the correct type (int)?
2. is the value in valid range (>=0)?
3. does the value depend on / have an interdependency with any other parameters (I guess not in this case)

Both should also be tested in tests. Also here I suggest to look at the tests for `validateFileUploadOptions`.

Reply:
I added validation for `undefined` and proper numeric type, but turned out I can't write unit tests to verify this behavior. The reason seems to be related to the fact that when we call to `reconfigureServer`, then, while the server restarts, `numberParse` function from `parsers.js` verifies that the argument is an integer. Therefore, it looks like the code in `Config.js` should assume that the configuration options already have the proper format and should only verify logical consistency.
Therefore, I removed all the additional logic and reverted to the existing implementation of this validation method.